app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+
+import { Formik, Form as FormikForm } from "formik";
+import { Check } from "neetoicons";
+import { Pane, Button, Toastr } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+
+import {
+  CONTACTS_FORM_INITIAL_FORM_VALUES,
+  CONTACTS_FORM_VALIDATION_SCHEMA,
+  ROLES,
+} from "../constants";
+
+const Form = ({ onClose }) => {
+  const [isSubmitted, setIsSubmitted] = useState(false);
+  const handleSubmit = () => {
+    try {
+      setIsSubmitted(true);
+      Toastr.success("Contact added SuccessFully");
+      onClose();
+    } catch (err) {
+      logger.error(err);
+    }
+  };
```

Comment:
Here I don't think you need a `try/catch` at the moment. From my understanding `onClose` doesn't `throw`.

Reply:
Ok will remove try-catch block